aspnetcore/signalr/security.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Security Considerations in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 05/01/2018
+uid: signalr/security
+---
+
+# Security Concepts in ASP.NET Core SignalR
```

Comment:
Should we talk about the buffer settings and the memory concerns if it's changed?

Reply:
YAS